src/unix/bsd/netbsdlike/netbsd/mod.rs

```diff
@@ -487,10 +487,6 @@ s! {
         af_arg: [[::c_char; 10]; 24],
     }
 
-    pub struct sched_param {
-        pub sched_priority: ::c_int,
-    }
-
```

Comment:
the struct definition should be moved to `src/unix/bsd/netbsdlike/mod.rs `, and not just removed.
else it is fine.

Reply:
Yes, I am sorry, didn't push that.